### PR TITLE
v0.3.3

### LINKED ISSUE #33: New release?
I've disabled the semihosting demo for thumbv8m-main.none-eabi in https://github.com/rust-embedded/cortex-m-rt/pull/182 until we can push out a release for this crate.

### PR DESCRIPTION
**Pull request by adamgreig**
Closes #33

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -7,7 +7,7 @@ license = "MIT OR Apache-2.0"
 name = "cortex-m-semihosting"
 readme = "README.md"
 repository = "https://github.com/japaric/cortex-m-semihosting"
```
Comment (therealprof) [inline Cargo.toml]:
```suggestion
repository = "https://github.com/rust-embedded/cortex-m-semihosting"
```

Can we change this, while we touch the stuff?

**Review by therealprof — APPROVED** [review]
LGTM

**Comment by adamgreig** [comment]
Published to crates.io and tagged.

**Pull request merged**